docker/hooks/push

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# push individual tags
+docker push "${IMAGE_NAME/:latest/:i386-latest}"
```

Comment:
This would need to support the actual tag being built, rather than latest, yeah?

Reply:
Correct. I don't know how your current DockerHub build setup looks like, so I left it vanilla.